roles/slurm/vars/ubuntu.yml

```diff
@@ -4,10 +4,10 @@ slurm_pam_lib_dir: /lib/x86_64-linux-gnu/security
 slurm_build_deps:
   - build-essential
   - libmunge-dev
-  - libmariadb-client-lgpl-dev
-  - libmysqlclient-dev
```

Comment:
I believe removing these packages is causing `slurmdbd` to fail on Ubuntu 18.04:

```
[2020-10-16T16:57:56.621] error: Couldn't find the specified plugin name for accounting_storage/mysql looking at all files
[2020-10-16T16:57:56.622] error: cannot find accounting_storage plugin for accounting_storage/mysql
[2020-10-16T16:57:56.622] error: cannot create accounting_storage context for accounting_storage/mysql
[2020-10-16T16:57:56.622] fatal: Unable to initialize accounting_storage/mysql accounting storage plugin
```

Reply:
Replacing `libmariadb-client-lgpl-dev` with `libmariadb-dev` for both Ubuntu 18.04 and 20.04 is OK. It looks like on 18.04 we should install `libmysqlclient-dev` and on 20.04 we should install `libmariadb-dev-compat`.

Reply:
Can confirm this works on both versions:

```
  - libmariadb-dev
  - libmariadbclient-dev
```